deepfence_worker/tasks/reports/data.go

```diff
@@ -118,7 +119,17 @@ func scanResultFilter(levelKey string, levelValues []string, masked []bool) repo
 
 	return filter
 }
+func CalculateOverallSeverityCounts(severityCountsList ...map[string]int32) map[string]int32 {
+	overallSeverityCounts := make(map[string]int32)
```

Comment:
nit: 
```suggestion
	overallSeverityCounts := map[string]int32{}
```
likewise everywhere else